workflow/rocoto/gfs_tasks.py

```diff
@@ -533,13 +533,13 @@ def aeroanlgenb(self):
     def aeroanlinit(self):
```

Comment:
The `aeroanlgenb` job (above this) should also replace `cdump` with `run` on line 512.  i.e.

```python
        dep_dict = {'type': 'task', 'name': f'{self.run}fcst'}
```

Reply:
And also line 517:
```python
        task_name = f'{self.run}aeroanlgenb'
```

Reply:
thanks, I missed those